logs_ingest/main.py

```diff
@@ -118,7 +118,7 @@ def deserialize_properties(record: Dict):
     properties_name = next((properties for properties in azure_properties_names if properties in record.keys()), "")
     properties = record.get(properties_name, {})
     if properties and isinstance(properties, str):
-        record["properties"] = json.loads(properties)
+        record["properties"] = parse_to_json(properties)
 
```

Comment:
Another thing with 
```
unparsed_timestamp
08/08/2022 17:48:23
```
`row 137  metadata_engine.apply(record, parsed_record)` - there is default rule in `config/default.json` for parsing timestamp and timestamp in a format `MM/dd/yyyy HH:mm:ss` is not parsed by log_forwarder. Please, investigate this behaviour as well.